bcda/main_test.go

```diff
@@ -243,51 +243,75 @@ func (s *MainTestSuite) TestCreateUser() {
 }
 
 func (s *MainTestSuite) TestCreateToken() {
-
-	// init
-	userUUID := "82503A18-BF3B-436D-BA7B-BAE09B7FFD2F"
-
 	// set up the test app writer (to redirect CLI responses from stdout to a byte buffer)
 	buf := new(bytes.Buffer)
 	s.testApp.Writer = buf
 
 	assert := assert.New(s.T())
-
-	// Test successful creation
-	args := []string{"bcda", "create-token", "--user-id", userUUID}
-	err := s.testApp.Run(args)
-	assert.Nil(err)
-	assert.NotNil(buf)
-	accessTokenString := strings.TrimSpace(buf.String())
-	assert.NotNil(accessTokenString)
-	buf.Reset()
-
-	// No parameters
-	args = []string{"bcda", "create-token"}
-	err = s.testApp.Run(args)
-	assert.Equal("User ID (--user-id) must be provided", err.Error())
-	assert.Equal(0, buf.Len())
-	buf.Reset()
-
-	// Blank User UUID
-	args = []string{"bcda", "create-token", "--user-id", ""}
-	err = s.testApp.Run(args)
-	assert.Equal("User ID (--user-id) must be provided", err.Error())
-	assert.Equal(0, buf.Len())
-	buf.Reset()
-
-	// Bad User UUID
-	args = []string{"bcda", "create-token", "--user-id", BADUUID}
-	err = s.testApp.Run(args)
-	assert.Equal("User ID must be a UUID", err.Error())
-	assert.Equal(0, buf.Len())
-	buf.Reset()
+	userUUID := "82503A18-BF3B-436D-BA7B-BAE09B7FFD2F"
+	clientID := "not_a_client"
+	clientSecret := "not_a_secret"
 
 	// Unexpected flag
-	args = []string{"bcda", "create-token", "--abcd", "efg"}
-	err = s.testApp.Run(args)
+	args := []string{"bcda", "create-token", "--abcd", "efg"}
+	err := s.testApp.Run(args)
 	assert.Equal("flag provided but not defined: -abcd", err.Error())
 	assert.Contains(buf.String(), "Incorrect Usage: flag provided but not defined")
+	buf.Reset()
+
+	switch auth.GetProvider().(type) {
+	case auth.OktaAuthPlugin:
```

Comment:
No need for a `case` statement here.  All the scenarios being tested in this switch are negative cases.  In order to increase test coverage, we should set the env var in the test case, execute tests, then set the env var back.  That way we get more test coverage.  There are lots of examples in our existing test suite where we override env vars and then set them back.

Reply:
Thanks!